TRACK_MAINTAINERS.md

```diff
@@ -0,0 +1,105 @@
+What this is about - TODO
+
+Information about who is a maintainer can be found on the [Exercism Teams
+page](https://github.com/orgs/exercism/teams) but unhelpfully, you need to be a
+team member before you can see the other team members.
+
+
+See also: 
+
+* https://github.com/exercism/discussions/issues/97
+* http://tinyletter.com/exercism/letters/exercism-behind-the-scenes-the-challenge-of-unmaintained-and-undermaintained-tracks
+
+
+# Instructions
+
+Sumbit a PR to this file add/remove maintainers.
+
+# Becoming a maintainer
+
+TODO
+
+# Stepping down as a matntainer
+
+TODO
+
+# Status
+
+ * ORPHANED - Nobody has merged anything in the past year.
+ * ENDANGERED - Somewhere between ORPHANED and AT RISK.
+ * AT RISK - Two people are actively discussing issues and reviewing and merging pull requests.
+ * MAINTAINED - Three or more people are actively discussing issues and reviewing and merging pull requests.
+
+### Orphaned
+
+* Bash
+* CoffeeScript
+* D
+* Groovy
+* Kotlin
+* PL/SQL
+* Perl 5
+* Pony
+* R
+* Standard ML
+
+### Endangered
+
+* Lua
+* Lisp
+* Erlang
+* Go
+* Factor
+* Perl 6
+* PHP
+* C++
+* Elm
+* Emacs Lisp
+* Lisp Flavored Erlang
+* MIPS Assembly
+* Racket
+* Scheme
+* Prolog
+
+### At Risk
+
+* Swift
+* Rust (but very active)
+* OCaml
+* JavaScript
+* ECMAScript
+* Python
+* Scala
+* F#
+* C#
+* Clojure
+* Objective-C
+
+### Maintained
+
+* Ruby   
+   Active: @kotp, @insti, @hilary, @tommyschaefer, @bmulvihill  
+   Alumni: @kytrynx, @bernardoamc, @Cohen-Carlisle  
+
+* C
+* Java
+* Crystal
+* Elixir
+* Haskell
+
+### Inactive Tracks
+
+There has not been enough activity to generate a heartbeat for the following tracks:
+
+* Ceylon
+* ColdFusion
+* Haxe
+* Idris
+* The Netwide Assembler
+* Nim
```

Comment:
Nim is for https://github.com/exercism/xnim (Nim should remain. If Vimscript is to be added, it should be added separately)